src/App.js

```diff
@@ -3,19 +3,21 @@ import './App.css';
 import { useState } from 'react'
 
 function App() {
-  const fiveContact = contactsArr.splice(0, 5);
-
+  const fiveContact = contactsArr.filter((contact,index)=> index<5);
+console.log(fiveContact)
```

Comment:
Alternative solution:

```
const fiveContacts = contactsArr.slice(0, 5);
```

Reply:
i used it before,,,but i came across lots of problem,,, then i changed it,,